lib/in-process-server-test-helpers.js

```diff
@@ -1,69 +1,27 @@
-/**
- * Helpers to run a Shields server in process.
- *
- * Usage:
- * let server;
- * before('Start running the server', function () {
- *   this.timeout(5000);
- *   server = serverHelpers.start();
- * });
- * after('Shut down the server', function () { serverHelpers.stop(server); });
- */
-
 'use strict'
 
-const config = require('./test-config')
 const serverConfig = require('./server-config')
-
-let startCalled = false
-
-/**
- * Start the server.
- *
- * @param {Number} port number (optional)
- * @return {Object} The scoutcamp instance
- */
-function start() {
-  if (startCalled) {
-    throw Error(
-      'Because of the way Shields works, you can only use this ' +
-        'once per node process. Once you call stop(), the game is over.'
-    )
+const Server = require('./server')
+
+function createTestServer({ port }) {
```

Comment:
I deferred handling defaults using Joi, though when that is implemented this can be done more cleanly, and likely the helper function can be removed.